test/run_all.py

```diff
@@ -68,15 +68,15 @@ def _suite():
 
 
 if __name__ == '__main__':
-    parser = optparse.OptionParser()
-    parser.add_option(
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
         '--log-level',
         '--log_level',
-        type='choice',
+        type=str,
```

Comment:
Could you explain why the type here is "str", and not six.text_type like it is elsewhere?

Reply:
Since the choices for this option is definitely within ascii code, using str on both python versions does not change the behavior. Hence, I thought importing six module and making this six.text_type not-so-efficient. If it is better to use the six module, I will fix this.

Reply:
I think I'd prefer to pay a small performance cost to be consistent with the --log-level parameters in other files. But either way is fine.

Reply:
I guess consistency is better in this situation.
Updated to use six.text_type